src/style.css

```
@@ -1 +1,81 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  background-color: white;
```

most of these are default
```suggestion
  display: flex;
```